app/views/site/about.html.erb

```diff
@@ -34,7 +34,6 @@
     <h2><div class='icon partners'></div><%= t ".partners_title" %></h2>
     <p><%= t 'layouts.partners_html',
              :ucl => link_to(t('layouts.partners_ucl'), "https://www.ucl.ac.uk"),
-             :ic => link_to(t('layouts.partners_ic'), "https://www.imperial.ac.uk/"),
```

Comment:
As previously discussed in #1944 it's not safe to remove IC like this as it will break every language that still has a `%{ic}` marker in the translation.

Reply:
I would suggest simply renaming layouts.partners_html in that case (the version without IC), not the most elegant solution, but in the grand scheme of things I don't think that really matters.